shared/chat/inbox/index.desktop.js

```diff
@@ -150,15 +147,22 @@ class Inbox extends PureComponent<Props, State> {
             onSelectDown={this._onSelectDown}
           />
           <NewConversation />
-          <div style={_scrollableStyle} onScroll={this._onScroll}>
-            <ReactList
-              ref={this._setRef}
-              useTranslate3d={false}
-              itemRenderer={this._itemRenderer}
-              length={this.props.rows.length}
-              type="variable"
-              itemSizeGetter={this._itemSizeGetter}
-            />
+          <div style={_listStyle}>
+            <AutoSizer>
+              {({height, width}) => (
+                <VariableSizeList
+                  height={height}
+                  width={width}
+                  ref={this._setRef}
+                  onItemsRendered={this._onItemsRendered}
+                  itemCount={this.props.rows.length}
+                  itemSize={this._itemSizeGetter}
+                  estimatedItemSize={56}
+                >
+                  {({index, style}) => this._itemRenderer(index, style)}
```

Comment:
I wonder if it would be even faster if we used PureComponent here. Probably a bit of work to move the itemRenderer out.

Reply:
No time is spent here basically